mica/version.py

```diff
@@ -14,7 +14,7 @@
 therein are applicable.
 """
 
-version = '3.12.1'
+version = '3.12.2'
```

Comment:
OK. I think this PR is reasonable as a bugfix so I've incremented the version accordingly.